.github/actions/demo/windows/action.yml

```diff
@@ -0,0 +1,29 @@
+name: windows-demo
+description: Build windows-demo for windows
+
+runs:
+  using: "composite"
+  steps:
+    - uses: extractions/setup-just@v1
+    - name: Install toolchain
+      shell: bash
+      run: just default-toolchain
+    - uses: Swatinem/rust-cache@v1
+    - uses: ilammy/msvc-dev-cmd@v1 # Provide access to lib.exe
+    - name: Install SQLite
+      shell: powershell
+      run: choco install sqlite -y --params "/NoTools"
+    - name: Build SQLite lib
+      shell: powershell
+      run: |
+        cd "C:\ProgramData\chocolatey\lib\SQLite\tools"
+        lib /DEF:sqlite3.def /OUT:sqlite3.lib /MACHINE:x64
+        echo "SQLITE3_LIB_DIR=C:\ProgramData\chocolatey\lib\SQLite\tools" >> $env:GITHUB_ENV
```

Comment:
That looks frightening :D

I think we could easily make sqlite an optional dependency. We only need sqlite in test builds actually. I'll create a PR tomorrow maybe.

Reply:
I struggled for a long time until I got it working. I thought about removing the dependency on rusqlite but I was not sure how. If you do, it will make the setup on Windows a little bit easier.

Reply:
So its very good that we have this working, as we will need it eventually. https://github.com/maplibre/maplibre-rs/issues/107

Reply:
Let's resolve this conversation as it works for now, we'll remove the sqlite3 step once #107 is done.